doc/source/ray-observability/state/ray-state-api-reference.rst

```diff
@@ -33,8 +33,11 @@ Log CLI
 Log CLI allows users to access the log from the cluster. 
 Note that only the logs from alive nodes are available through this API.
 
-.. click:: ray.scripts.scripts:ray_logs
-    :prog: ray logs
+.. click:: ray.experimental.state.state_cli:log_file
+    :prog: ray logs file
+
+.. click:: ray.experimental.state.state_cli:log_actor
+    :prog: ray logs actor
```

Comment:
We also need ray logs worker <Worker ID OR PID>

Reply:
Maybe we can do it in a follow up

Reply:
But we don't have `--worker-id` yet? Is this working already?